superset/charts/dao.py

```diff
@@ -70,15 +71,15 @@ def overwrite(slc: Slice, commit: bool = True) -> None:
             db.session.commit()
 
     @staticmethod
-    def favorited_ids(charts: List[Slice], current_user_id: int) -> List[FavStar]:
+    def favorited_ids(charts: List[Slice]) -> List[FavStar]:
```

Comment:
Will we ever need to get a list of favorited charts for other users?

Reply:
@ktmud no. Also this logic is going to refactored in another PR. In general Superset does not handle users other than the current logged in user. There have been recent mechanisms added to override the user globally if needed.